Silicon/NXP/iMX6Pkg/AcpiTables/Dsdt-Common.inc

```diff
@@ -0,0 +1,112 @@
+/** @file

```

Comment:
Seems like new lines are not normalized. Some files have that ^M and some don't

Reply:
For some reason, this file is the only one that shows up with ^M.  These are CRLF endings which is the expected style for EDK2.  All other files also have CRLF endings.  Leaving it as is.